lib/debugging.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Functions to aid debugging, such as for saving screenshots and logs.
+ */
+import * as path from 'path';
+import {driver} from './index';
+import {getEnabledLogTypes, saveLogs} from './logs';
+
+/**
+ * Adds an afterEach() hook to the current test suite to save logs and screenshots after failed
+ * tests. These are saved only if `MOCHA_WEBDRIVER_LOGDIR` variable is set, and named:
```

Comment:
The idea is that you'd call this to make it available, but when running manually or debugging, you don't need these things logged. You'd set LOGDIR for running in, say, Jenkins, so that you have a record to look at when tests fail. Having a warning/error when running manually would be annoying, I think.